### PR TITLE
-deleted-

### LINKED ISSUE #12845: Migration: add Cancel button
currently you you have to go to `<gitea-url>/<owner>/<migrating-repo>/settings` and use the "Delete Repo" button, to cancel migration

A cancel Button would be nice

### PR DESCRIPTION
**Pull request by 6543**
close #12845

### CODE REVIEW

File: routers/repo/setting.go [hunk 1]
```diff
@@ -54,6 +55,28 @@ func Settings(ctx *context.Context) {
...
+		ctx.Error(http.StatusNotFound)
+		return
+	}
+
+	if ctx.Repo.Repository.Status != models.RepositoryBeingMigrated {
+		ctx.Error(http.StatusConflict, "repo already migrated")
+		return
+	}
+
+	if err := repo_service.DeleteRepository(ctx.User, ctx.Repo.Repository); err != nil {
```
Comment (lunny) [inline routers/repo/setting.go:75]: We should send a stop singal to the migration go routine if it's migrating.

**Comment by mrsdizzie** [comment]
Add a screenshot ?

**Comment by 6543** [comment]
picture upload on gh dos not work currently

**Comment by 6543** [comment]
Have a github bug: push to this branch wont do anything ...
... close this pull and open new one

**Comment by 6543** [comment]
-> #12917

**Pull request closed without merging**